de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractIncomingProjectNegotiation.java

```diff
@@ -220,6 +232,16 @@ protected void cleanup(IProgressMonitor monitor,
         Map<String, IProject> projectMapping) {
         fileReplacementInProgressObservable.replacementDone();
 
+        /*
+         * TODO Move disable queuing responsibility to SarosSession (see todo
+         * in {@link ArchiveIncomingProjectNegotiation#transfer}).
+         */
+        for (IProject project : projectMapping.values())
```

Comment:
So the TODO links to an inherited class, however every class can inherit from this class. Also this will log warnings when there was no queuing enabled.


The Javadoc is also not helpful for the cleanup method.